plugins/inputs/webhooks/github/github_webhooks_models.go

```diff
@@ -107,10 +106,7 @@ func (s CommitCommentEvent) NewMetric() telegraf.Metric {
 		"commit":  s.Comment.Commit,
 		"comment": s.Comment.Body,
 	}
-	m, err := metric.New(meas, t, f, time.Now())
-	if err != nil {
-		log.Fatalf("Failed to create %v event", event)
-	}
+	m, _ := metric.New(meas, t, f, time.Now())
```

Comment:
why don't we want to error check these?

Reply:
so actually those metric.new methods don't actually have any possibility of returning an error. in the future we should refractor the hundreds of uses of it to handle removing that error possibility, but for now it could be ignored.